recipes/auditwheel/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: auditwheel
+  version: "1.1.0"
+
+source:
+  fn: auditwheel-1.1.0.tar.gz
+  url: https://pypi.python.org/packages/source/a/auditwheel/auditwheel-1.1.0.tar.gz
+  md5: f42ec0ed1ce5e09e5ef5d3ce4f9826e6
+
+build:
+  skip: True  # [not linux]
```

Comment:
auditwheel requires [Python 3.3+](https://github.com/pypa/auditwheel#installation) so adding `skip: True  # [py2k]` would probably be reasonable.   

I checked the dependency on my system, auditwheel will install in Python 2.7, but raises a SyntaxError when `auditwheel` is executed.